xml/deployment_additional.xml

```diff
@@ -68,36 +68,73 @@ if necessary.
   </important>
 
   <para>
-   To install the &kube; dashboard, perform the following steps:
+   Install the &kube; dashboard. 
   </para>
 
+  <note>
+   <title>Requirements</title>
+    <para>
+      Heapster version 1.3.0 or later must be installed to the cluster.  
+      helm instructions to install the upstream version are included in this section. 
+    </para>
+  </note>
+
+  <note>
+   <title>Tool versions</title>
+    <para>
+      helm version 2.7.2+ and kubectl version 1.8.0+ recommended
+    </para>
+  </note>
+
   <procedure>
    <title>Installation of &kube; Dashboard</title>
+   <step>
+     <screen>
+helm install --name heapster-default --namespace=kube-system stable/heapster --version=0.2.7 --set rbac.create=true
+     </screen>
+   </step>
+   <step>
+     <screen>
+helm install --namespace=kube-system --name=kubernetes-dashboard stable/kubernetes-dashboard --version=0.6.1
+     </screen>
+   </step>
    <step>
     <para>
-     Connect to the &admin_node; using SSH.
+      SSH into any cluster node. 
     </para>
    </step>
    <step>
     <para>
-     Run the command:
+      Visit <literal>http://127.0.0.1:8081/</literal> in your browser. 
+      You should be greeted with a Welcome page. There is one Configure dialog with 2 options, <literal>kubeconfig</literal> and <literal>token</literal> authentication. 
     </para>
-<screen>kubectl create -f https://raw.githubusercontent.com/kubernetes/dashboard/v1.6.3/src/deploy/kubernetes-dashboard.yaml</screen>
    </step>
    <step>
     <para>
-     After the previous command finishes, run the following to expose the
-     proxy:
+      Select <literal>token</literal> authentication. To retrieve your token refer to the value in your kubeconfig file by running the command:
     </para>
-<screen><command>kubectl proxy</command>
-    Starting to serve on 127.0.0.1:8001</screen>
+    <screen>
+grep "id-token" kubeconfig  | awk '{print $2}'
+    </screen>
+   </step>
+  </procedure>
+
+  <para>
+  On login cluster resources and basic metrics are populated. 
+  </para>
+
+  <procedure>
+   <title>Exposing the Dashboard</title>
+   <step>
+<screen>helm upgrade kubernetes-dashboard stable/kubernetes-dashboard --set service.type=NodePort
+</screen>
    </step>
    <step>
     <para>
-     In your browser open <emphasis>http://127.0.0.1:8001/api/v1/proxy/namespaces/kube-system/services/kubernetes-dashboard/#!/workload?namespace=default</emphasis>. The
-     &kube; dashboard is running there.
+      Visit <literal>https://WORKER_NODE_ADDRESS:NODE_PORT</literal> in your browser.
```

Comment:
Visit &lt;literal>https://&lt;replaceable>WORKER_NODE_ADDRESS&lt;/replaceable>:&lt;replaceable>NODE_PORT&lt;/replaceable>&lt;/literal> in your browser.

* Why visit this address?
* What to do once I have opened the page in the browser?
* Does this have to be done on each worker node?

Reply:
Is there a document somewhere that explains what replaceable is? I compared the output in my generated PDF and it appeared the same, so I assumed <literal> worked like HTML <pre>

Reply:
This section is just breaking out exposing the dashboard from the rest of the setup, because not every customer might want that. I can add a clarification that this URL is for access from outside of the cluster though.